test/studies/bale/indexgather/ig-auto-agg.ml-execopts

```diff
@@ -1,17 +1,4 @@
-#!/usr/bin/env python3
-
-# Run bale index gather with 1 million updates per task. Use a smaller problem
-# size for non-aggregated configurations with low small-messages rates.
-import os
-
-comm = os.getenv('CHPL_COMM')
-comm_sub = os.getenv('CHPL_COMM_SUBSTRATE')
-ugni = comm == 'ugni'
-gn_aries = comm == 'gasnet' and comm_sub  == 'aries'
-
-N  = 10000
-NL = 1000000
-if ugni or gn_aries:
-  N = NL
-
-print('--N={0} --printStats --mode=Mode.ordered    # bale-ig-auto-aggregated'.format(N))
+# other versions of this test chooses N based on the comm layer and the version
+# of the benchmark kernel. However, in auto/manual aggregation we always use
+# larger N. See ig.ml-execopts
+--N=1000000 --printStats --mode=Mode.ordered    # bale-ig-auto-aggregated'.format(NL))
```

Comment:
Trailing/leftover `'.format(NL))`

Reply:
I don't know what to say. Thanks for catching it.